CONTRIBUTING.md

```diff
@@ -36,6 +36,9 @@ composed of the following members:
 You can reach TSC at [eve-tsc@lists.lfedge.org](mailto:eve-tsc@lists.lfedge.org)
 and also [browse the archives](https://lists.lfedge.org/g/eve-tsc).
 
+You can signup for the [eve-tsc mailing list at](https://lists.lfedge.org/g/eve-tsc/)
+and also susbscribe to the [eve-tsc Calendar at](https://lists.lfedge.org/g/eve-tsc/calendar)
```

Comment:
There are two calendars; the other is https://lists.lfedge.org/g/eve/calendar
I see that some events appear on both but it makes sense listing both of them here.